frontend/Makefile

```diff
@@ -2,7 +2,6 @@
 # stdin in the continuous build.
 
 all:
-	npm install -g pnpm
```

Comment:
This is not possible on NixOS; and is the only "breaking" change I've made here. This just needs to be do manually, now, by non-nix users.